File: Bars.Gkh.Overhaul.Hmao/Entities/CostLimit.cs
using Bars.B4.DataAccess;
using Bars.Gkh.Entities;
using Bars.Gkh.Entities.Dicts;
using System;

namespace Bars.Gkh.Overhaul.Hmao.Entities
{
    /// <summary>
    /// Предельная стоимость услуги
    /// </summary>
    public class CostLimit : BaseEntity
    {
        /// <summary>
        /// Работа
        /// </summary>
        public virtual Work Work { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public virtual Municipality Municipality { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public virtual decimal Cost { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public virtual DateTime? DateStart { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public virtual DateTime? DateEnd { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public virtual short? FloorStart { get; set; }

        /// <summary>
        /// 
        /// </summary>
        public virtual short? FloorEnd { get; set; }
    }
}

File: Bars.Gkh.Overhaul.Hmao/libs/B4/model/CostLimit.js
Ext.define('B4.model.CostLimit', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'CostLimit'
    },
    fields: [
        { name: 'Id' },
        { name: 'Work' },
        { name: 'Cost' },
        { name: 'DateStart' },
        { name: 'DateEnd' },
        { name: 'FloorStart' },
        { name: 'FloorEnd' },
        { name: 'Municipality' },
    ]
});